packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -44,13 +49,27 @@ const init = ({ mainWindow, store }: Dependencies) => {
     let isManualCheck = false;
     let latestVersion: Partial<UpdateInfo> = {};
     let updateCancellationToken: CancellationToken;
+    let shouldInstallUpdateOnQuit = false;
+
     const updateSettings = store.getUpdateSettings();
 
+    // update is downloaded on click and applied automatically whenever user restarts application (now or later)
     autoUpdater.autoDownload = false;
-    autoUpdater.autoInstallOnAppQuit = true;
+    // manually force install based on `shouldInstallUpdateOnQuit` var instead to have more control over the process
+    autoUpdater.autoInstallOnAppQuit = false;
```

Comment:
Why do we need `more control over the process`? Is it because the electron-updater is unreliable and we believe our handling will work better? Or something else? Either way, it would be cool to explain the motivation in the comment.

Reply:
@tsusanka should I?